tests/utils.go

```diff
@@ -2669,7 +2669,7 @@ func WaitUntilVMIReady(vmi *v1.VirtualMachineInstance, expecterFactory VMIExpect
 
 	// Fetch the new VirtualMachineInstance with updated status
 	virtClient, err := kubecli.GetKubevirtClient()
-	vmi, err = virtClient.VirtualMachineInstance(NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
+	vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
```

Comment:
If you decided to fix one, then you should fix all (for all Get and Delete). Having only one fixed make no sense.
(in this file)

If you want to do it, I suggest you do it on a separate PR to avoid other surprises.

Reply:
i have to fix this one, because i use WaitUntilVMIReady for non default namespace.
I dont think we need to fix every occurrence on this PR ?
and even if so, lets keep this one in this PR if possible please and fix all the others on other PR?
(else i must first create a PR fixing all, and then use it in this PR)

Reply:
Then squash it as part of fixing the existing tests... If you take this to its own commit, from mys point of view you should fix them all.

Reply:
ok will then squash it (or put as part of the fixing tests), and split the last commit in other PR as you suggested, thanks

Reply:
i will then squash it as part of the refactor in the dual stack separate PR, based on the last commit and this commit,
since i needed it because the refactor.